book/src/pong-tutorial/pong-tutorial-03.md

```diff
@@ -156,15 +164,51 @@ impl<'s> System<'s> for PaddleSystem {
 # fn main() {}
 ```
 
-Now lets add this system to our `GameDataBuilder` in `main.rs`:
+Now, there's quite a bit going on here!
+
+We create a unit struct `PaddleSystem`, and implement the `System` trait for it
+with the lifetime of the components on which it operates.
```

Comment:
Impl lifetime elision (`'_`)? I don't think that will help since it needs to be named to be referenced in multiple places. Or is there something else you're thinking about?

Reply:
ehh I believe I was thinking of a fantasy blending NLL and lifetime elision, so I understood wrong :zipper_mouth_face:. I must admit that lifetimes are something I try to avoid unless the compiler lectures me about it. 

That said, I do agree that the part `[...] with the lifetime of the components on which it operates.` should either be removed or get complementary explanation. 
Since I don't know `System` and the rest of the engine well enough yet, I'm more in favour of removing that part and leaving it up to someone else to make a subchapter / paragraph with a more elaborate explanation. Or leave the text as-is.

Reply:
For clarity before closing: I will leave this paragraph as-is, as I feel the tutorial could benefit from an ulterior addition regarding lifetimes in the engine... in another PR :smiley: